app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const { getCategories } = require('./controllers/app.controllers');
+const app = express();
+
+app.use(express.json())
```

Comment:
We don't need this yet because we're not doing a post request yet and it's good practise to only include code that is needed for the ticket 🙂